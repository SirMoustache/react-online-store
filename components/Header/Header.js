/**
 * Absolute imports
 */
import React from 'react';
import styled from 'styled-components';

const Root = styled.header`
  flex: 0 0 auto;
`;

const Header = () => <Root>Header placeholder</Root>;

export default Header;
