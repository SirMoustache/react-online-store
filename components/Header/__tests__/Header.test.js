/**
 * Absolute imports
 */
import React from 'react';
import { shallow } from 'enzyme';

/**
 * Local Components
 */
import Header from '..';

describe('<Header />', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<Header />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
