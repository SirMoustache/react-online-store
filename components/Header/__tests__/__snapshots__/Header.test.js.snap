// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should match snapshot 1`] = `
<Header__Root>
  <Header />
</Header__Root>
`;
